src/main/scala/gpp/ui/forms/EnumViewMultipleSelect.scala

```diff
@@ -53,9 +53,10 @@ final case class EnumViewMultipleSelect[F[_], A](
   fluid:                js.UndefOr[Boolean] = js.undefined,
   header:               js.UndefOr[ShorthandS[VdomNode]] = js.undefined,
   icon:                 js.UndefOr[ShorthandS[Icon]] = js.undefined,
+  id:                   js.UndefOr[String] = js.undefined,
   inline:               js.UndefOr[Boolean] = js.undefined,
   item:                 js.UndefOr[Boolean] = js.undefined,
-  label:                js.UndefOr[ShorthandS[String]] = js.undefined,
+  label:                js.UndefOr[String] = js.undefined,
```

Comment:
Good catch

Reply:
There are some instances in scalajs-rreact-semantic-ui, as well. I'm going to put in a PR there to switch the relevant ones to `ShorthandS[Label]`. Then, we could make these the same. FormInput already is setup that way.